monitor.cpp

```diff
@@ -5,34 +5,45 @@
 #include <iostream>
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if (temperature > 102 || temperature < 95) {
-    cout << "Temperature is critical!\n";
-    for (int i = 0; i < 6; i++) {
+void alertBreak() {
+  int repeatCount = 6;
+  int timeInSeconds = 1;
+  for (int i = 0; i < repeatCount; i++) {
       cout << "\r* " << flush;
-      sleep_for(seconds(1));
+      sleep_for(seconds(timeInSeconds));
       cout << "\r *" << flush;
-      sleep_for(seconds(1));
+      sleep_for(seconds(timeInSeconds));
     }
+}
+
+int temperatureOK(float temperature) {
+  if (temperature > 102 || temperature < 95) {
```

Comment:
Try isolating the range-comparison into a function, instead of repeating the comparison. Treat it as a conversion from an input-value to a message